webapi/src/test/scala/org/knora/webapi/responders/admin/SipiResponderADMSpec.scala

```diff
@@ -20,43 +20,55 @@
 package org.knora.webapi.responders.admin
 
 import akka.testkit._
+import com.typesafe.config.{Config, ConfigFactory}
 import org.knora.webapi._
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectRestrictedViewSettingsADM
 import org.knora.webapi.messages.admin.responder.sipimessages._
-import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
 import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
 
 import scala.concurrent.duration._
 
-object SipiResponderADMSpec {
-
-    // A test UserProfileV1.
-    private val requestingUser: UserADM = SharedTestDataADM.incunabulaMemberUser
-
-    private val fileValueResponseFull = SipiFileInfoGetResponseADM(permissionCode = 6)
+object SipiResponderV1Spec {
```

Comment:
This looks like a merge error. The responder is now `SipiResponderADM`, so the spec is called `SipiResponderADMSpec`.

Reply:
changed